algo-coding-exercise/backtest/src/test/java/codingblackfemales/backtest/AddCancelAlgoBackTest.java

```diff
@@ -117,16 +120,17 @@ public void testExampleBackTest() throws Exception {
         //create a sample market data tick....
         send(createSampleMarketDataTick());
         //simple assert to check we had 3 orders created
-        //assertEquals(container.getState().getChildOrders().size(), 3);
+        assertEquals(container.getState().getChildOrders().size(), 21);
```

Comment:
Does this also run?

Reply:
Yes, I'm gonna revert this file back to how I met it. I was trying to figure out how the test works when I wrote that.